numpy/ma/core.pyi

```diff
@@ -995,7 +1000,28 @@ mean: _frommethod
 nonzero: _frommethod
 prod: _frommethod
 product: _frommethod
-ravel: _frommethod
+
+# Keep in sync with `np.ravel`
+@overload
+def ravel(self: _ArrayLike[_ScalarT_co], /, order: _OrderKACF = "C") -> MaskedArray[tuple[int], dtype[_ScalarT_co]]: ...
```

Comment:
Might as well use the the alias:

```suggestion
def ravel(a: _ArrayLike[_ScalarT], order: _OrderKACF = "C") -> _MaskedArray1D[_ScalarT]: ...
```

Also note that covariance is only relevant to generic type-*parameters*, not to type-*arguments* like here).